parm/archive/gdas_restarta.yaml.j2

```diff
@@ -32,6 +32,8 @@ gdas_restarta:
         - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}abias_int"
         - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}dtfanl.nc"
         - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}loginc.txt"
+        {% else %}
+        - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}radbcor"
```

Comment:
I think this is a good opportunity to 1) add the extension to the filename so we know it's a tarball and 2) give it a slightly longer name that is more transparent.

What about: `gdas.t00z.rad_varbc_params.tar`?
At minimum, I think we should do the `gdas.t00z.radbcor.tar` as I know several times new people are tripped up by what this file is (a tar ball of files)

Reply:
Good suggestion @CoryMartin-NOAA .  I'm fine with using a more descriptive name for the radiance bias correction tarball, `gdas.t*z.rad_varbc_params` or `gdas.t*z.rad_varbc_coeffs` seem reasonable, but let me tag @ADCollard and @emilyhcliu to get their take on what they would like to see.

Can we use the `.tar.` suffix in operations?  The operational GSI cnvstat, oznstat, and radstat files are tarballs with no suffix.  I'm fine with adding the suffix `.tar` if NCO's implementation standards permit this.  Do you know @aerorahul if the suffix `.tar` OK for operational files which are tarballs?

Reply:
> Good suggestion @CoryMartin-NOAA . I'm fine with using a more descriptive name for the radiance bias correction tarball, `gdas.t*z.rad_varbc_params` or `gdas.t*z.rad_varbc_coeffs` seem reasonable, but let me tag @ADCollard and @emilyhcliu to get their take on what they would like to see.
> 
> Can we use the `.tar.` suffix in operations? The operational GSI cnvstat, oznstat, and radstat files are tarballs with no suffix. I'm fine with adding the suffix `.tar` if NCO's implementation standards permit this. Do you know @aerorahul if the suffix `.tar` OK for operational files which are tarballs?

1. A more descriptive name for the tar file is great.
2. Adding an extension so we know it is a tar file is also good.

Reply:
@KateFriedman-NOAA and @WalterKolczynski-NOAA , please pause any work planned for g-w issue #2869.  The DA team has not reached consensus on the name to use for the radiance bias correction tarball.

Reply:
Execute `find -O3 -L /lfs/h1/ops/prod/com/ -name "*tar"` on Cactus.  The `find` command returned 14567 instances of the suffix `.tar`.  Give this, rename radiance bias correction tarball file as `${prefix}.rad_varbc_params.tar`.   Done at 73aa233f.